docs/deprecated/index.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
This page is not showing up for me in the preview. Perhaps change the file name to docs/deprecated/deprecation-listing.md